profiles/opentelemetry_dev/README.md

```diff
@@ -0,0 +1,19 @@
+# opentelemetry_dev
+
+## Usage
+
+*Add a description detailing what this profile does and any extra instructions for using it.*
+
+## Extra Variables
+
+*List any extra variables that user's can configure in their .compose.env*
+
+- `MY_VAR_1`
+    - Description: My custom variable.
+    - Options:
+        - 1: do thing 1.
+        - 2: do thing 2.
+    - Default: 1
+- `MY_VAR_2`
+    - Description: path to the file that does the thing.
+    - Default: Unset
```

Comment:
I miss some details about the usage and purpose of this profile. Would you mind adding notes to this file?

Reply:
I added a basic README to it @lubosmj.
We can improve it as we're using on a daily basis.